build/vega-lite-schema.json

```diff
@@ -124,6 +124,166 @@
         }
       ]
     },
+    "AnyMarkConfig": {
+      "anyOf": [
+        {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/AreaConfig"
+            },
+            {
```

Comment:
Why are there these "not"s?  Kinda weird.

Reply:
https://github.com/vega/ts-json-schema-generator/issues/344. It's pretty hairy and I'll solve it later.